tools/templates/acrbuild.yml

```diff
@@ -45,3 +45,10 @@ jobs:
       azureSubscription: azureiiot
       scriptLocation: inlineScript
       inlineScript: powershell ./tools/scripts/acr-build.ps1 $(dockerFolder)
+  - task: AzureCLI@1
```

Comment:
Master had this, Should this be removed?

Reply:
No, let's keep it. The task makes sure that we have debug images.